### PR TITLE
provide samples for kotlin.text any() extension function KT-35867

### PR DESCRIPTION
**Pull request by hamzeh-pourshabanan**
Addresses [KT-35867](https://youtrack.jetbrains.com/issue/KT-35867) for the any() function.

### CODE REVIEW

File: libraries/stdlib/samples/test/samples/text/strings.kt [hunk 1]
```diff
@@ -507,4 +507,29 @@ class Strings {
 
         assertPrints(mixedColor, "brown&blue")
     }
+
+    @Sample
+    fun all() {
+        val name = "fatima"
+        val containsWhiteSpace: Boolean = name.any{
```
Comment (qurbonzoda) [inline libraries/stdlib/samples/test/samples/text/strings.kt:514]: Could we use `assertPrints(name.any { it.isWhitespace() }, false)` directly?

File: libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt [hunk 2]
```diff
@@ -31,14 +31,22 @@ object Aggregates : TemplateGroupBase() {
...
+        specialFor(CharSequences) {
+            sample("samples.text.Strings.all")
+        }
 
         doc {
             """
             Returns `true` if all ${f.element.pluralize()} match the given [predicate].
             """
         }
-        sample("samples.collections.Collections.Aggregates.all")
```
Comment (qurbonzoda) [inline libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt:41]: After changing the template files sources should be regenerated using `kotlin:libraries:tools:kotlin-stdlib-gen:run` Gradle task.

**Review by qurbonzoda — CHANGES_REQUESTED** [review]

File: libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt (hunk 2 above)
Comment (hamzeh-pourshabanan) [reply libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt:41]: Thank you for your review @qurbonzoda.
All requested changes has been done and template files sources have been regenerated.

File: libraries/stdlib/samples/test/samples/text/strings.kt [hunk 3]
```diff
@@ -507,4 +507,18 @@ class Strings {
 
         assertPrints(mixedColor, "brown&blue")
     }
+
+    @Sample
+    fun all() {
+        assertPrints("fatima".any(Char::isWhitespace), "false")
+        assertPrints("Ariya".all(Char::isLowerCase), "false")
```
Comment (qurbonzoda) [inline libraries/stdlib/samples/test/samples/text/strings.kt:514]: Let's have here two strings that have opposite properties. e.g:
```
assertPrints("London".any(Char::isWhitespace), "false")
assertPrints("Rio de Janeiro".any(Char::isWhitespace), "true")

assertPrints("London".all(Char::isLetter), "true")
assertPrints("Rio de Janeiro".all(Char::isLetter), "false")
```
In my opinion, it better shows how `all` and `any` differs. What do you think?

File: libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt [hunk 4]
```diff
@@ -31,14 +31,22 @@ object Aggregates : TemplateGroupBase() {
...
         inline()
-        specialFor(ArraysOfUnsigned) { inlineOnly() }
+        specialFor(ArraysOfUnsigned) {
+            inlineOnly()
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(Maps) {
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(CharSequences) {
```
Comment (qurbonzoda) [inline libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt:41]: This should be enough:
```
sample("samples.collections.Collections.Aggregates.all")
specialFor(CharSequences) {
    sample("samples.text.Strings.all")
}
```
should be enough.

The DSL means that _"samples.collections.Collections.Aggregates.all" is the sample for all receivers, but for CharSequences there is a different sample_

So basically the second `sample(...)` overrides the previous.

File: .idea/runConfigurations.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (qurbonzoda) [inline .idea/runConfigurations.xml:1]: Could you please exclude this local and unrelated change from the Pull Request?

File: kotlin-native/runtime/src/main/kotlin/generated/_ArraysNative.kt [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
```
Comment (qurbonzoda) [inline kotlin-native/runtime/src/main/kotlin/generated/_ArraysNative.kt:2]: Please rebase onto a fresh master, the updated copyright is already there.

**Review by qurbonzoda — CHANGES_REQUESTED** [review]

**Comment by fzhinkin** [comment]
Closing the PR as inactive. Feel free to reopen it.
